### DISCUSSION TITLE
Port Game To Vulkan

### DISCUSSION THREAD

**Discussion by 733737** [OpenRCT2/OpenRCT2#17737] [Feature requests]
Adding Vulkan support to openrct2 may sound extremely time consuming and exhausting, but it's been done in other source ports before. GZDoom has Vulkan renderer support and so do some other source ports and open source projects. I don't know if anyone is interested or has experience, or if there would be any benefit whatsoever to porting it

**Comment by duncanspumpkin** [comment]
Feel free to make a PR adding support for it.
